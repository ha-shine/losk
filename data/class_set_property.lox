class Person {}

var personA = Person();
var personB = Person();
personA.other = personB;
personB.name = "Shine";

print Person;
print personA;
print "Hello, " + personA.other.name;