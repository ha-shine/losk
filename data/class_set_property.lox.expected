<Class Person>
<Instance Person.class>
Hello, Shine
